packages/api/src/services/security/auth-providers/external-strategy.ts

```diff
@@ -0,0 +1,33 @@
+import {AuthStrategyType} from '../../../models/security/authentication';
+import {SecurityContextService} from '../security-context.service';
+import {service} from '../../../providers';
+import {BaseGdbLoginStrategy} from './base-gdb-login-strategy';
+
+export class ExternalStrategy extends BaseGdbLoginStrategy {
```

Comment:
We are using `Provider` insted of strategy.

I am ok with both, but be consistent.

Reply:
I have opened a task for this